Runtimes/Core/runtime/CMakeLists.txt

```diff
@@ -2,6 +2,20 @@ configure_file("CMakeConfig.h.in"
   "${PROJECT_BINARY_DIR}/include/swift/Runtime/CMakeConfig.h"
   ESCAPE_QUOTES @ONLY)
 
+install(FILES
+  "${PROJECT_BINARY_DIR}/include/swift/Runtime/CMakeConfig.h"
+  DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/swift/Runtime"
+  COMPONENT SwiftCore_development)
+
+add_library(swiftRuntimeCMakeConfig INTERFACE)
+target_include_directories(swiftRuntimeCMakeConfig INTERFACE
+  $<$<COMPILE_LANGUAGE:C,CXX>:$<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include/>$<INSTALL_INTERFACE:$<INSTALL_PREFIX>/${CMAKE_INSTALL_INCLUDEDIR}>>
+  $<$<COMPILE_LANGUAGE:Swift>:$<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include/>$<INSTALL_INTERFACE:$<INSTALL_PREFIX>/${CMAKE_INSTALL_INCLUDEDIR}>>)
```

Comment:
I don't think that we should be using this as a library within this module itself.